doc/modules/multiclass.rst

```diff
@@ -14,52 +14,116 @@ Multiclass and multilabel algorithms
+
+  For example, prediction of weight, sugar content and number of calories using
+  features extracted from images of fruit. Each sample is one image of a fruit
+  and for each sample weight, sugar content and number of calories are all
+  output.
+
+  - valid representation :term:`multilabel` `y` is:
+
+    - dense matrix of shape ``(n_samples, n_classes)`` of floats. A column wise
+      concatenation of :term:`continuous` variables.
+
+
+- **Multioutput-multiclass classification**
+  (also known as **multitask classification**):
+  classification task which labels each sample with a set of **non-binary**
+  attributes. Both the number of attributes and the number of classes per
+  attribute is greater than 2. A single estimator thus handles several joint
+  classification tasks. This is both a generalization of the multi\ *label*
+  classification task, which only considers binary attributes, as well as a
+  generalization of the multi\ *class* classification task, where only one
+  attribute is considered.
+
+  For example, classification of the attributes "type of fruit" and "colour"
+  for a set of images of fruit. The attribute "type of fruit" has the possible
+  classes: "apple", "pear" and "orange". The attribute "colour" has the
+  possible classes: "green", "red", "yellow" and "orange". Each sample is an
+  image of a fruit, a label is output for both attributes and each label is
+  one of the possible classes of the corresponding attribute.
+
+  - valid representation :term:`multilabel` `y` is:
+
+    - dense matrix of shape ``(n_samples, n_classes)`` of floats. A column wise
+      concatenation of 1d :term:`multiclass` variables.
+
+  - Note that any classifiers handling multioutput-multiclass
+    (also known as multitask classification) tasks,
+    support the multilabel classification task as a special case.
+    Multitask classification is similar to the multioutput
+    classification task with different model formulations. For
+    more information, see the relevant estimator documentation.
+
 
 All scikit-learn classifiers are capable of multiclass classification,
 but the meta-estimators offered by :mod:`sklearn.multiclass`
 permit changing the way they handle more than two classes
 because this may have an effect on classifier performance
 (either in terms of generalization error or required computational resources).
 
+**Summary**
+
++-----------------+-------------+-------------+------------------------------------------+
+|                 | Number of   | Classes per | Valid                                    |
+|                 | attributes  | attribute   | :func:`~utils.multiclass.type_of_target` |
++=================+=============+=============+==========================================+
+| Multiclass      |  1          | >2          | - 'multiclass'                           |
+| classification  |             |             | - sparse binary                          |
```

Comment:
Not sure that having the output of `type_of_target` and the type of `y` on the same column is intuitive

Reply:
Removed.